src/entities/catalogs/services/update-catalog-channels.ts

```
@@ -35,28 +33,20 @@ export async function updateCatalogChannels(
+ * @returns An array of channel metadata objects with ID, handle, title, description and logo
  *
  * @remarks
  * Fetches channel information from the YouTube API and transforms the response into a structured format.
- * Extracts key details such as channel title, description, custom URL, thumbnail, and associated topics.
+ * Extracts key details such as channel title, description, custom URL and thumbnail.
  *
  * @throws {Error} If there's an issue fetching channel information from the YouTube API
  */
 async function getChannelsInfo(channels: string[]) {
-  let channelsInfo: any[] = [];
+  let channelsInfo: CatalogChannel[] = [];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'channelsInfo' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)